example.env

```diff
@@ -82,6 +82,10 @@ DKIM_PUBLIC_KEY_PATH=local_data/dkim.pub.key
 # Local SQLite database
 DB_URI=sqlite:///db.sqlite
 # Postgres
+# POSTGRES_USER=sluser
```

Comment:
These lines can be removed right?

Reply:
In fact, there's already `?Required` flag in `docker-compose.yml` (for those variables to be specified) so that the user can't start the services without setting up this information. I supposed it's better for security?

Reply:
Yeah it's better this way indeed 👌